source/_integrations/tedee.markdown

```diff
@@ -48,10 +48,74 @@ Local Access Token:
   type: string
 {% endconfiguration_basic %}
 
+## Supported devices
+
+This integration supports
+
+- Tedee PRO
+- Tedee GO
+
 ## Binary sensors
 
-We have three binary sensors: One that indicates whether the battery is currently charging, one indicating if the pull spring is enabled, and one indicating whether the lock is in a "semi-locked" position, meaning the lock has been turned manually and between its normal end positions.
+- **Charging**: indicates whether the battery is currently charging.
+- **Pullspring enabled**: indicates whether the pull spring setting is enabled.
+- **Semi locked**: indicates whether the lock is in a "semi-locked" position. "Semi-locked" means the lock has been turned manually and is between its normal end positions. The lock itself will be unavailable in this position.
```

Comment:
oh, I think I missed one. Semi-locked and uncalibrated are UI strings?

Reply:
Wdym with "UI-strings"?

Reply:
![image](https://github.com/user-attachments/assets/60fc9477-fd7d-40f1-9448-def01a3cd9e3)

Reply:
A string shown in the user interface. We mark them bold.

Reply:
Probably "yes"

Reply:
thx! :pray:

Reply:
Was going to mark them bold, but I was mistaken. the "" are an explanation, not used as UI string. apologies for the confusion.